common/lib/xmodule/xmodule/assets/vertical/public/js/vertical_student_view.js

```diff
@@ -13,4 +24,32 @@ window.VerticalStudentView = function(runtime, element) {
             apiUrl: $bookmarkButtonElement.data('bookmarksApiUrl')
         });
     });
+    $(element).find('.vert').each(
+        function(idx, block) {
+            var blockKey = block.dataset.id;
+
+            if (block.dataset.completableByViewing === undefined) {
+                return;
+            }
+            // TODO:
```

Comment:
@jcdyer: Thanks.   Can you please change this first line to when you squash?:
```
// TODO: EDUCATOR-1778
```
In the future, I may ask you to use "TODO: EDUCATOR-1778" so we can track blockers to enabling for edx.org.  Thank you!

Reply:
Great idea.